modules/rostests/kmtests/example/Example_user.c

```diff
@@ -26,9 +27,15 @@ START_TEST(Example)
     KmtRunKernelTest("Example");
 
     /* now start the special-purpose driver */
-    KmtLoadDriver(L"Example", FALSE);
+    Error = KmtLoadDriver(L"Example", FALSE);
+    ok_eq_int(Error, 0);
```

Comment:
0 --> ERROR_SUCCESS (here and in the other changes)

Reply:
OK, I changed it and I somewhat regret it, because it was simply a waste of time. Everyone with a brain understands 0 == ERROR_SUCCESS.